centos.sh

```diff
@@ -102,49 +104,49 @@ generate_new_ramdisk() {
     # pick the latest kernel
     VERSION="$(ls -r -1 $FOLD/hdd/boot/vmlinuz-* | head -n1 |cut -d '-' -f 2-)"
```

Comment:
My bad: `for file in "$FOLD/hdd/boot/vmlinuz-"*; do VERSION="${file#*-}"; done` the asterisk must not be part of the quoted string